client/src/components/MoreOptions.tsx

```diff
@@ -1,11 +1,20 @@
-export default function MoreOptions() {
+import { Link } from 'react-router-dom';
+import { Entry } from '../lib/types';
+type props = {
+  entry: Entry;
+};
+
+export default function MoreOptions({ entry }: props) {
   return (
     <div className="z-10 bg-white rounded-lg drop-shadow-lg w-44 absolute">
       <ul className="py-2 text-sm text-gray-700 dark:text-gray-200">
         <li>
-          <span className="block px-4 py-2 hover:bg-gray-100 dark:hover:bg-gray-600 dark:hover:text-white">
+          <Link
+            to={'/edit-adventure'}
+            state={{ entry }}
```

Comment:
Ah, nice! I've had to tell other students that. It's awesome you found that on your own.